src/shell/commands/executable.rs

```diff
@@ -34,8 +40,13 @@ impl ShellCommand for ExecutableCommand {
           Ok(std::env::current_exe()?)
         }) {
           Ok(command_path) => command_path,
-          Err(err) => {
-            let _ = stderr.write_line(&err.to_string());
+          Err(ExecutableCommandError::CommandNotFound(err)) => {
+            let _ = stderr.write_line(&format!("{}: command not found", err));
```

Comment:
Nitpick: we could just use `command_name` from above here and change `CommandNotFound` to not have a string associated with it. I got a bit confused reading `err` here for a second`.

Reply:
Changed it :+1: